app/controllers/application_controller.rb

```diff
@@ -1,2 +1,15 @@
 class ApplicationController < ActionController::Base
+  before_action(:load_current_player)
+  before_action(:force_player_sign_in)
+  
+  def load_current_player
+    @current_player = Player.where({ :id => session[:player_id] }).at(0)
```

Comment:
I note that we're using `[]` on `session` here, as opposed to our patch of `.fetch`; yet we're using our patch of `.at(0)` on `ActiveRecord::Relation`, rather than `.first`.

I could buy the argument of either dropping the dependency on our patches and going with `[]` and `.first`; or explicitly including the patches along with the `draft_generator` gem (although that would require some thinking about whether we need to include the gem in `production` group, or generate the required intializers).

But I think we should pick one for consistency. For now, I think the simpler road would be to go with `[]`/`.first`; particularly since this code will most likely remain untouched, and just read. What do you think?

Reply:
@raghubetina I actually forgot we patched `session`. I think it's okay to make the assumption that this generator will only be used in the AppDev world. I think in the future when all the patches are made into one appdev gem we re-visit how to handle production cases.